axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveLineControlServiceImpl.java

```diff
@@ -111,7 +111,7 @@ public void validateMoveLine(MoveLine moveLine) throws AxelorException {
 
     if (CollectionUtils.isNotEmpty(moveLine.getInvoiceTermList())) {
       List<InvoiceTerm> invoiceTermList = moveLine.getInvoiceTermList();
-      Invoice invoiceAttached = invoiceTermList.get(0).getInvoice();
+      Invoice invoiceAttached = moveLine.getMove().getInvoice();
```

Comment:
You need to find another way to fix this, we need to keep `invoiceTermList.get(0)` and also because in another comment I ask you to remove the invoice you set on the Move, so this fix won't work anymore.